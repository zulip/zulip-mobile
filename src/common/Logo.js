/* @flow strict-local */
import React from 'react';
import { Image } from 'react-native';

import logoImg from '../../static/img/logo.png';
import { createStyleSheet } from '../styles';

const styles = createStyleSheet({
  logo: {
    width: 40,
    height: 40,
    margin: 20,
    alignSelf: 'center',
  },
});

export default () => <Image style={styles.logo} source={logoImg} resizeMode="contain" />;
