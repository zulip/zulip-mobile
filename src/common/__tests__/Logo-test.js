import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '../Logo';

it('renders correctly', () => {
  const tree = renderer.create(
    <Logo />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
