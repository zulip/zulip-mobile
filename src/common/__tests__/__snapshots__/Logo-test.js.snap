exports[`test renders correctly 1`] = `
<Image
  resizeMode="contain"
  source={1}
  style={
    Object {
      "alignSelf": "center",
      "height": 100,
      "width": 100
    }
  } />
`;
